cgmes/cgmes-model/src/main/resources/CIM16.sparql

```diff
@@ -738,7 +739,7 @@ WHERE {
         cim:GeneratingUnit.minOperatingP ?minP ;
         cim:GeneratingUnit.maxOperatingP ?maxP ;
         cim:GeneratingUnit.initialP ?initialP ;
-        cim:IdentifiedObject.name ?name .
+        cim:IdentifiedObject.name ?nameGeneratingUnit .
```

Comment:
Not linked to this PR, but this will be useful when aliases will be a working feature